src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -853,86 +871,53 @@ private Request GetSuccessfulRequest(IAnalyzerOptions analyzerOptions)
             return request;
         }
 
-        private static void AssertMessageLogged(Mock<ILogger> loggerMock, string message)
+        private static VCConfiguration CreateVCConfigurationWithProperties(string platformName, IDictionary<string, string> propertyMap = null)
         {
-            loggerMock.Verify(x => x.WriteLine(It.Is<string>(
-                s => s.Equals(message))), Times.Once);
-        }
+            var vcPlatformMock = new Mock<VCPlatform>();
+            vcPlatformMock.SetupGet(x => x.Name).Returns(platformName);
 
-        private static void AssertPartialMessageLogged(Mock<ILogger> loggerMock, string message)
-        {
-            loggerMock.Verify(x => x.WriteLine(It.Is<string>(
-                s => s.Contains(message))), Times.Once);
-        }
+            var vcConfigMock = new Mock<VCConfiguration>();
+            vcConfigMock.SetupGet(x => x.Platform).Returns(vcPlatformMock.Object);
 
-        private class DummyProjectEngine : IVCRulePropertyStorage
```

Comment:
We're no longer using reflection/dynamic code to fetch properties, so we can just use normal Mocks of the real product interfaces.